website/content/docs/concepts/client-count.mdx

```diff
@@ -108,20 +107,26 @@ From a Vault client perspective, **_option 1_** is one client, the Master CI/CD
 The number of active clients using a Vault cluster is the total of:
 
 - active entities: identity entities that create a token via a login
-- active non-entity tokens: tokens created via a method that is not associated with an entity
+- active non-entity tokens: clients associated with tokens created via a method
```

Comment:
```suggestion
- active non-entity clients: clients associated with tokens created via a method
```

Reply:
So here I purposefully kept this as `active non-entity tokens`, the reason for this being that the API will currently display these as `non_entity_tokens` . Thus, I wanted the documentation to convey essentially that "the active non-entity tokens count maps to clients associated with tokens, as opposed to just tokens".